contracts/src/tags/index.ts

```diff
@@ -0,0 +1,196 @@
+import { initContract } from '@ts-rest/core';
+import {
+  subDefaultTagQuerySchema,
+  subDefaultTagResponseSchema,
+  superDefaultTagResponseSchema,
+  superTagIdQuerySchema,
+  subTagResponseSchema,
+  tagsOfBookResponseSchema,
+  modifySuperTagBodySchema,
+  modifyTagResponseSchema,
+  incorrectTagFormatSchema,
+  alreadyExistTagSchema,
+  defaultTagCannotBeModifiedSchema,
+  modifySubTagBodySchema,
+  NoAuthorityToModifyTagSchema,
+  mergeTagsBodySchema,
+  invalidTagIdSchema,
+  createTagBodySchema,
+  duplicateTagSchema,
+  tagIdSchema,
+} from './schema';
+import {
+  badRequestSchema,
+  bookInfoIdSchema,
+  bookInfoNotFoundSchema,
+  forbiddenSchema,
+  paginationQuerySchema,
+  serverErrorSchema,
+} from '../shared';
+
+const c = initContract();
+
+export const tagContract = c.router(
+  {
+    getSubDefault: {
+      method: 'GET',
+      path: '',
+      summary: '서브/디폴트 태그 정보를 검색한다.',
+      description: '서브/디폴트 태그 정보를 검색한다. 이는 태그 관리 페이지에서 사용한다.',
+      query: subDefaultTagQuerySchema,
+      responses: {
+        200: subDefaultTagResponseSchema,
+        400: badRequestSchema,
+        401: forbiddenSchema,
+        500: serverErrorSchema,
+      },
+    },
+    getSuperDefaultForMain: {
+      method: 'GET',
+      path: '/main',
+      summary: '메인 페이지에서 사용할 태그 목록을 가져온다.',
+      description: '슈퍼 태그(노출되는 태그), 디폴트 태그(노출되지 않고 분류되지 않은 태그)를 랜덤한 순서로 가져온다. 이는 메인 페이지에서 사용된다.',
+      query: paginationQuerySchema.omit({ page: true }),
+      responses: {
+        200: superDefaultTagResponseSchema,
+        400: badRequestSchema,
+        401: forbiddenSchema,
+        500: serverErrorSchema,
```

Comment:
```suggestion
        401: forbiddenSchema,
```
400번 500번대 오류는 생략해도 괜찮을 것 같아요.

400 (올바르지 않은 입력): 프론트엔드 contracts에서 zod로 이미 검증하고 있어서
500 (서버 오류): 모든 API 요청에 공통적으로 나오기 때문에

Reply:
앗 맞네요, 그러면 400부터 500번대는 tag의 다른 api에서도 생략해서 다시 PR 올리겠습니다 :)